recipes/chemfiles-lib/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.6.0" %}
+{% set sha256 = "4e43dbe1b1d4c3a2eae923fecc3452d2b207efc1d511f6683164d51d22b0255f" %}
+
+package:
+  name: chemfiles-lib
+  version: {{ version }}
+
+about:
+  home: http://chemfiles.github.io
+  license: MPL-v2.0
+  summary: Modern library for chemistry file reading and writing
+  doc_url: http://chemfiles.github.io/chemfiles/latest/
+  dev_url: https://github.com/chemfiles/chemfiles
+
+build:
+  script:
+    - export MACOSX_DEPLOYMENT_TARGET=""  # [osx]
```

Comment:
I do not remember, but I got an error on Os X when building the recipe a few months ago. I'll check again.